pepys_import/resolvers/command_line_resolver.py

```diff
@@ -479,17 +486,16 @@ def fuzzy_search_platform(
             )
         elif choice in completer:
             # Extract the platform details from the string
-            name_or_xgraph, identifier, nationality = choice.split(" / ")
+            platform_details = choice.split(" / ")
+            name, identifier, nationality = (
+                platform_details[0],
+                platform_details[-2],
+                platform_details[-1],
+            )
             # Get the platform from the database
             platform = (
                 data_store.session.query(data_store.db_classes.Platform)
```

Comment:
@BarisSari - we used to perform an "or" operation here, the successive `filter` calls now look more like an "and".

Could you describe why we're changing our strategy please?

Reply:
Sure. `or` operation was used because some rows didn't have platform name. So, platforms were filtered by trigraph or quadgraph when rows without a platform name were selected. But right now, it is not possible to select a line which doesn't have a platform name. That's why I removed it.